subjects/printandmiss/README.md

```diff
@@ -0,0 +1,46 @@
+## printandmiss
+
+### Instructions
+
+Write a function called `PrintAndMiss` that takes a string and an integer and return a string containing the characters until reaching the integer, then skipping that same number of characters, and repeats until the end of the string.
+
+- If the string is empty return `Invalid Output` followed by newline `\n`.
+- If the integer is `0` or it's negative return the string followed by a newline `\n`.
+
+
+### Expected function
+
+```go
+func PrintAndMiss(arg string, num int) string {
+}
+```
+### Usage
+
+Here is a possible program to test your function :
+
+```go
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Print(PrintAndMiss("123456789", 3))
+ 	fmt.Print(PrintAndMiss("abcdefghijklmnopqrstuvwyz", 3))
+ 	fmt.Print(PrintAndMiss("", 3))
+ 	fmt.Print(PrintAndMiss("hello you all ! ", 0))
+ 	fmt.Print(PrintAndMiss("what is your name?", 0))
```

Comment:
just remove this example here, i guess have the case with  this one : 
```go
fmt.Print(PrintAndMiss("hello you all ! ", 0))
```
both have the same value in the second param.